x-pack/test/cases_api_integration/security_and_spaces/tests/trial/connectors/cases/cases_connector.ts

```diff
@@ -0,0 +1,1321 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type SuperTest from 'supertest';
+import { createHash } from 'node:crypto';
+import stringify from 'json-stable-stringify';
+import {
+  CasesConnectorRunParams,
+  OracleRecordAttributes,
+} from '@kbn/cases-plugin/server/connectors/cases/types';
+import { AttachmentType, CasePostRequest } from '@kbn/cases-plugin/common';
+import {
+  AlertAttachment,
+  Attachments,
+  Case,
+  CaseStatuses,
+  CaseSeverity,
+  ConnectorTypes,
+  CustomFieldTypes,
+} from '@kbn/cases-plugin/common/types/domain';
+import { KbnClient } from '@kbn/test';
+import { CasePersistedAttributes } from '@kbn/cases-plugin/server/common/types/case';
+import {
+  SEVERITY_EXTERNAL_TO_ESMODEL,
+  STATUS_EXTERNAL_TO_ESMODEL,
+} from '@kbn/cases-plugin/server/common/constants';
+import { Client } from '@elastic/elasticsearch';
+import { ALERTING_CASES_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
+import { User } from '../../../../../common/lib/authentication/types';
+import {
+  globalRead,
+  noKibanaPrivileges,
+  onlyActions,
+} from '../../../../../common/lib/authentication/users';
+import {
+  deleteAllCaseItems,
+  executeConnector,
+  findCases,
+  getAllComments,
+  updateCase,
+  getCase,
+  getConfigurationRequest,
+  createConfiguration,
+  createComment,
+} from '../../../../../common/lib/api';
+import { getPostCaseRequest, postCommentAlertReq } from '../../../../../common/lib/mock';
+import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
+import { roles as api_int_roles } from '../../../../../../api_integration/apis/cases/common/roles';
+import {
+  casesAllUser,
+  obsCasesAllUser,
+  obsCasesReadUser,
+  obsSecCasesAllUser,
+  obsSecCasesReadUser,
+  secAllCasesReadUser,
+  secAllSpace1User,
+  secAllUser,
+  users as api_int_users,
+} from '../../../../../../api_integration/apis/cases/common/users';
+import { createUsersAndRoles, deleteUsersAndRoles } from '../../../../../common/lib/authentication';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Case connector', () => {
+    const connectorId = 'system-connector-.cases';
```

Comment:
Shouldn't this be just `.cases`?

Reply:
The actions framework generates the connector's ID automatically by pretending `system-connector-` to all system connectors.